workspace/viewer/PdfJsViewer.ts

```diff
@@ -444,8 +448,10 @@ export class PdfJsViewer {
     return true
   }
 
-  private positionCanvas() {
-    const canvas = $(".canvasWrapper", this.jqRoot);
+  private positionCanvas(pageNumber: number) {
+    const parent = $("#pageContainer" + pageNumber, this.jqRoot);
```

Comment:
There is nifty [string interpolation](https://basarat.gitbooks.io/typescript/docs/template-strings.html) in TypeScript

Reply:
Done.